src/App.tsx

```diff
@@ -15,36 +16,106 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
+  SortAlphabet = 'alphabet',
+  SortLength = 'length',
+  Default = '',
+}
+
+interface FilterParams {
+  isReversed: boolean;
+  sortField: SortType;
+}
+
+function getPreparedGoods(
+  goods: string[],
+  { sortField, isReversed }: FilterParams,
+) {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
```

Comment:
There is no need to check sortField. If we don't pass sortField the ```default``` case will work

Reply:
> There is no need to check sortField. If we don't pass sortField the `default` case will work

But you actually don't need to sort if it's not passed, so the check is a good idea